pkg/mons/restore_quorum.go

```diff
@@ -278,13 +279,15 @@ func getMonDetails(goodMon string, monEndpoints []string) ([]string, string, str
 	return badMons, goodMonPublicIp, goodMonPort, nil
 }
 
-func promptToContinueOrCancel(answer string) (string, error) {
+func PromptToContinueOrCancel(inputSource, expectedAnswer, answer string) (string, error) {
 	if skip, ok := os.LookupEnv("ROOK_PLUGIN_SKIP_PROMPTS"); ok && skip == "true" {
 		return "skipped prompt since ROOK_PLUGIN_SKIP_PROMPTS=true", nil
 	}
 
-	if answer == "yes-really-restore" {
-		return "proceeding", nil
+	if inputSource == "restore-quorum" && answer == expectedAnswer {
```

Comment:
The input source shouldn't matter when comparing the answer
```suggestion
	if answer == expectedAnswer {
```

Reply:
right, removed